examples/s/d/nn/xcase/realization-0/iter-0/rms/bin/export_a_surface.py

```diff
@@ -13,15 +13,17 @@
 
 CFG = ut.yaml_load("../../fmuconfig/output/global_variables.yml")
 
-
-INPUT_FOLDER = Path("../output/maps/props")
-FILE = "poro_average.gri"
+FILES = {
+    "poro": Path("../output/maps/props") / "poro_average.gri",
+    "depth": Path("../output/maps/structure/") / "topvolantis--ds_extract_geogrid.gri",
+}
 
 
 def main():
     """Exporting maps from clipboard"""
 
-    surf = xtgeo.surface_from_file(INPUT_FOLDER / FILE)
+    print("Export a porosity average map")
+    surf = xtgeo.surface_from_file(FILES["poro"])
```

Comment:
Exporting maps from clipboard? Looks like a file

Reply:
I believe the idea here was to emulate a file being exported from the somewhere inside RMS. A "poro" surface existed from before. In this PR, I have expanded that to also include a depth surface to feed the aggregation example.

Reply:
Clarified a bit better in the docstring that this is an emulation of an export